### DISCUSSION TITLE
Fix for 'Run gem install bundler' error

### DISCUSSION THREAD

**Discussion by sarah-beasley** [open-sdg/open-sdg#2080] [Announcements]
This error may be coming up when trying to make a change in your site repository:
![image](https://github.com/open-sdg/open-sdg/assets/106670682/df8e26af-4837-40b4-9265-b3d5747f082c)

Please make the below changes in order to fix this:
In your .github/workflows folder in your **site** repository - you will need to update the "gem install bundler" line in all of the files that you have in this folder that contain this line, to point to a specific version.

For example, in the following files in your .github/workflows folder:
deploy-to-production.yml 
deploy-to-staging.yml 
test-pull-requests.yml

You need to replace:

`gem install bundler`

with: 

`gem install bundler -v 2.4.22`

**Comment by christoph-LSN** [comment]
@sarah-beasley 
Perhaps it's helpfull to add that the .github/workflows folder of the **site** repository containing the gem **install bundler** line

**Reply by sarah-beasley** [reply]
Thanks Christoph
